spec/views/volunteer_ops/popup.html.erb_spec.rb

```diff
@@ -16,7 +16,7 @@
 
   before(:each) do
     allow(VolunteerOp).to receive(:where).and_return([op1, op2])
-    render partial: "popup.html.erb", locals: {org: org}
+    render partial: "popup_harrow.html.erb", locals: {org: org}
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)